repo/packages/R/riak/1/config.json

```diff
@@ -0,0 +1,204 @@
+{
+    "type": "object",
+    "properties": {
+        "riak": {
+            "type": "object",
+            "description": "Riak Mesos Framework specific properties",
+            "properties": {
+                "framework-name": {
+                    "type": "string",
+                    "description": "Framework Instance Name.",
+                    "default": "riak"
+                },
+                "hostname": {
+                    "default": "riak.marathon.mesos",
+                    "type": "string",
+                    "description": "Framework HTTP API hostname."
+                },
+                "marathon": {
+                    "type": "string",
+                    "description": "The Marathon URL.",
+                    "default": "marathon.mesos:8080"
+                },
+                "master": {
+                    "type": "string",
+                    "description": "The URL of the Mesos master.",
+                    "default": "leader.mesos:5050"
+                },
+                "zk": {
+                    "type": "string",
+                    "description": "Zookeeper address. Specify in the format host:port.",
+                    "default": "leader.mesos:2181"
```

Comment:
yup, with the port: 
`zk-1.zk:2181,zk-2.zk:2181,zk-3.zk:2181,zk-4.zk:2181,zk-5.zk:2181`